open-api-spec.yaml

```diff
@@ -451,6 +435,181 @@ paths:
           in: query
           name: cursor
           description: The cursor key to list from
+  /quotes:
+    post:
+      summary: Create a Quote
+      tags: []
+      operationId: create-quote
+      responses:
+        '201':
+          description: Quote Created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/quote'
+              examples:
+                New Fixed Send Quote for $25:
+                  value:
+                    id: 'https://openpayments.guide/alice/quotes/8c68d3cc-0a0f-4216-98b4-4fa44a6c88cf'
+                    accountId: 'https://openpayments.guide/alice/'
+                    receivingAccount: 'https://openpayments.guide/bob/'
+                    sendAmount:
+                      value: '2500'
+                      assetCode: USD
+                      assetScale: 2
+                    receiveAmount:
+                      value: '2198'
+                      assetCode: EUR
+                      assetScale: 2
+                    amountType: 'fixed-send'
+                    createdAt: '2022-03-12T23:20:50.52Z'
+                    updatedAt: '2022-03-12T23:20:50.52Z'
+                    expiresAt: '2022-04-12T23:20:50.52Z'
+        '401':
+          $ref: '#/components/responses/401'
+        '403':
+          $ref: '#/components/responses/403'
+      requestBody:
+        content:
+          application/json:
+            examples:
+              Create fixed-send-amount quote for $25:
+                value:
+                  receivingAccount: 'https://openpayments.guide/alice'
+                  sendAmount:
+                    value: '2500'
+              Create fixed-receive-amount quote for $25:
+                value:
+                  receivingPayment: 'https://openpayments.guide/alice/incoming-payments/37a0d0ee-26dc-4c66-89e0-01fbf93156f7'
+                  receiveAmount:
+                    value: '2500'
+            schema:
+              type: object
+              properties:
+                receivingAccount:
```

Comment:
Why does the receiving account matter for the quote?

Reply:
As with outgoing payments, the client may specify either `receivingAccount` or `receivingPayment`.
When `receivingAccount` is specified, an outgoing payment will create an incoming payment (`receivingPayment`), but a quote will not.

Reply:
The quote needs a [destination](https://github.com/interledgerjs/interledgerjs/tree/master/packages/pay#quoteoptions) (a [ResolvedPayment](https://github.com/interledgerjs/interledgerjs/tree/master/packages/pay#resolvedpayment)), either an account or an incoming payment.

Reply:
Of course :see_no_evil: